multinode-demo/gce_multinode.sh

```diff
@@ -0,0 +1,90 @@
+#!/bin/bash
+
+command=
+prefix=
+num_nodes=
+out_file=
+image_name="ubuntu-16-04-cuda-9-2-new"
+
+usage() {
+  exitcode=0
+  if [[ -n "$1" ]]; then
+    exitcode=1
+    echo "Error: $*"
+  fi
+  cat <<EOF
+usage: $0 <-c create/delete> <-p prefix> <-n num_nodes> <-o file> [-i image-name]
+
+Creates a GCE multinode network
+
+ -c create/delete - Create or delete the network
```

Comment:
Can you make the command an argument instead of an option? So you'd write `gce_multinode.sh create` instead of `gce_multinode.sh -c create`?

Reply:
Let me try